specs/slider/slider.md

```diff
@@ -0,0 +1,189 @@
+# slider + slider-label

```

Comment:
Prefer we stick to titles formatted in uppercase so "Slider + Slider Label" - see tree view spec.